src/functions.stub.php

```diff
@@ -3,26 +3,38 @@
 /** @generate-function-entries */
 
 namespace MongoDB\BSON {
+    /** @deprecated */
```

Comment:
You can add a hint for fixing the deprecation:
```suggestion
    /** @deprecated use MongoDB\BSON\Document::fromPHP() instead  */
```

Reply:
Good point, added these for all functions except `toJSON`, which doesn't have a replacement.